### DISCUSSION TITLE
Welcome to esci Discussions!

### DISCUSSION THREAD

**Discussion by rcalinjageman** [rcalinjageman/esci#13] [Announcements]
This is a discussion space for esci.  You can use this space to post questions, point out problems, request features, etc.  It's a bt more low-key than creating a formal issue in github, and comments posted here can easily be elevated into formal issues.

Cheers,

Bob

**Comment by gdcumming** [comment]
This Geoff. Great to see this discussion space--please do leave comments, plaudits, links to teaching materials, grumps, ....
